packages/cli/commands/create/api-sample.js

```diff
@@ -1,7 +1,5 @@
-const {
-  createApiSamplePrompt,
-  overwriteSamplePrompt,
-} = require('../../lib/createApiSamplePrompt');
+const { createApiSamplePrompt } = require('../../lib/createApiSamplePrompt');
+const { folderOverwritePrompt } = require('../../lib/createApiSamplePrompt');
```

Comment:
Looks like this lives in `../../lib/prompts` now.

Reply:
Fixed